src/tty-ask-password-agent/tty-ask-password-agent.c

```diff
@@ -313,6 +324,40 @@ static int wall_tty_block(void) {
         return fd;
 }
 
+static int parse_filter(const char *arg) {
+        const char *l;
+        bool strict = false;
```

Comment:
redundant?

Reply:
removed this one in favor of parse_pid :+1: